### PR TITLE
Repo skeleton

### PR DESCRIPTION
**Pull request by nsklikas**
IAM-999

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* metadata.yaml > published on Charmhub
+* documentation > published on (or linked to from) Charmhub
+* detailed contribution guide > documentation or CONTRIBUTING.md
+
+Use links instead.
+-->
+
+# ldap-integrator
+
+Charmhub package name: operator-template
```
Comment (natalian98) [inline README.md:13]:
```suggestion
Charmhub package name: ldap-integrator
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+More information: https://charmhub.io/ldap-integrator
+
+Describe your charm in one or two sentences.
+
+## Other resources
+
+<!-- If your charm is documented somewhere else other than Charmhub, provide a link separately. -->
+
+- [Read more](https://example.com)
```
Comment (natalian98) [inline README.md:22]: I know it's just a skeleton and will be changed soon but perhaps you could link the identity platform docs instead: https://charmhub.io/topics/canonical-identity-platform

**Review by natalian98 — COMMENTED** [review]
lgtm, just some small suggestions

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      stages: [commit-msg]
+      args: [--strict]
+- repo: https://github.com/igorshubovych/markdownlint-cli
+  rev: v0.41.0
+  hooks:
+  - id: markdownlint
+- repo: https://github.com/renovatebot/pre-commit-hooks
+  rev: 37.385.0
+  hooks:
+    - id: renovate-config-validator
```
Comment (wood-push-melon) [inline .pre-commit-config.yaml:48]: Don't forget to remove this "noisy" thing.

File: renovate.json [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    {
+      "groupName": "testing deps",
+      "matchFiles": ["tox.ini"],
+      "matchUpdateTypes": ["major", "minor", "patch", "pin", "digest"],
+      "automerge": true,
+      "schedule": ["on monday"],
+      "additionalBranchPrefix": "auto-"
+    },
+    {
+      "groupName": "renovate packages",
```
Comment (wood-push-melon) [inline renovate.json:59]: Don't forget to remove this.

File: ldap-integrator_ubuntu-22.04-amd64.charm
[the remark this answers is not in the document]
Comment (nsklikas) [reply ldap-integrator_ubuntu-22.04-amd64.charm:1]: yep, removed

File: README.md (hunk 2 above)
Comment (nsklikas) [reply README.md]: changed

File: .pre-commit-config.yaml (hunk 3 above)
Comment (nsklikas) [reply .pre-commit-config.yaml]: done

**Review by natalian98 — APPROVED** [review]

**Pull request merged**